src/app/about/About.module.css

```diff
@@ -0,0 +1,30 @@
+.about {
+    /* styles */
+    text-align: center;
+}
+
+.about__image_container {
```

Comment:
[Nit]: As this is in `About.module.css`, there's no need to use an `about__` prefix. Also, prefer `kebab-case` for HTML/CSS identifiers (`id`/`class`) as with URLs (not fully sure the origin for this though).